### ISSUE TITLE
ImportError when trying to run bot

### ISSUE THREAD

**Issue by Daddypaddy777** [AmericanPresidentJimmyCarter/yal-discord-bot#8]
ImportError: cannot import name 'LlamaConfig' from 'transformers' (/home/user/yal-discord-bot/env/lib/python3.10/site-packages/transformers/__init__.py)

**Comment by AmericanPresidentJimmyCarter** [comment]
What version of transformers do you have installed?

**Comment by Daddypaddy777** [comment]
transformers-4.22.1

**Comment by Daddypaddy777** [comment]
Updated to transformers-4.29.0.dev0 and now when I run "python setup_cuda.py install" I get the following: 
"The detected CUDA version (12.1) mismatches the version that was used to compile
PyTorch (11.7). Please make sure to use the same CUDA versions."

If I ignore that and proceed with running the bot I get:
"RuntimeError: Error(s) in loading state_dict for LlamaForCausalLM:
        Missing key(s) in state_dict:"
Followed by a whole bunch of model. layers. Any ideas?

**Comment by Daddypaddy777** [comment]
May have something to do with CUDA extension?

**Comment by AmericanPresidentJimmyCarter** [comment]
Yes, it does. Your CUDA version installed must match the version used to compile the pytorch you currently have installed.

**Comment by AmericanPresidentJimmyCarter** [comment]
> If I ignore that and proceed with running the bot I get:
> "RuntimeError: Error(s) in loading state_dict for LlamaForCausalLM:
> Missing key(s) in state_dict:"
> Followed by a whole bunch of model. layers. Any ideas?

This is caused by using old versions of the weights that are now outdated. The ones detailed in the README are recommended.
